src/SfxWeb/src/app/views/node/essentials/essentials.component.html

```diff
@@ -36,37 +42,5 @@ <h4>Deactivation Info</h4>
     </app-collapse-container>
   </div>
 
-  <div class="detail-pane essen-pane" *ngIf="repairJobs.length > 0" data-cy="repair-jobs">
-    <app-collapse-container sectionName="Repair Jobs">
-      <div collapse-header>
-        <h4>Repair jobs</h4>
-      </div>
-      <div collapse-body>
-        <app-detail-list *ngIf="node" [list]="repairJobs"
-          [listSettings]="repairJobSettings"></app-detail-list>
-      </div>
-    </app-collapse-container>
-  </div>
-
-  <div class="detail-pane essen-pane">
-    <app-collapse-container sectionName="Unhealthy Evaluations">
-      <div collapse-header>
-        <h4>Health Evaluations</h4>
-      </div>
-      <div collapse-body>
-          <app-health-viewer *ngIf="node" [healthyEvaluations]="node.health.healthEvents" [unhealthyEvaluations]="node.health.unhealthyEvaluations" ></app-health-viewer>
-      </div>
-    </app-collapse-container>
-  </div>
-
-  <div class="detail-pane essen-pane" *ngIf="deployedApps">
-    <app-collapse-container sectionName="Deployed Applications">
-      <div collapse-header>
-        <h4>Deployed Applications</h4>
-      </div>
-      <div collapse-body>
-        <app-detail-list [list]="deployedApps.collection" [listSettings]="listSettings" data-cy="appsList">
-        </app-detail-list>
-      </div>
-    </app-collapse-container>
-  </div>
+  <a class="mif-info link mif-lg" routerLink="./events" style="float: right;" title="More Events"></a>
+  <app-events [isShown]="false"></app-events>
```

Comment:
this should be wrapped in a collapse container.

Reply:
We want to make the events table a collapsible container?

Reply:
Yes, we do this for consistency because it can also take up a fair amount of space so it should be possible to get it out of view.

Reply:
Gotcha. That makes sense